AdobeIms/Test/Unit/Model/LogOutTest.php

```diff
@@ -97,6 +97,9 @@ public function testExecute(): void
     {
         $this->userContextInterfaceMock->expects($this->exactly(1))
             ->method('getUserId')->willReturn(1);
+        $this->userProfileInterfaceMock->expects($this->once())
+            ->method('getAccessToken')
```

Comment:
Hello @sivaschenko ,
Why we need to test the `getAccessToken` method ? As it's not present in `AdobeIms/Model/LogOut.php`. Sorry, Correct me If I'm wrong.
Thanks

Reply:
This method is called in `AdobeIms/Model/LogOut.php` to retrieve a token for Logout URL generation

Reply:
Yes, You are right. Thanks for the clarification :)